src/components/firebase-support.ts

```diff
@@ -39,3 +39,19 @@ export const subscribeFcmNotifications = (convoClient: Client) => {
       console.log("FcmNotifications: can't request permission", err);
     });
 };
+
+export const showNotification = (pushNotification: PushNotification) => {
+  const notificationTitle = "Notification: " + pushNotification.type;
+  const notificationOptions = {
+    body: pushNotification.body,
+    icon: "favicon.ico",
+  };
+
+  const notification = new Notification(notificationTitle, notificationOptions);
+  notification.onclick = function (event) {
```

Comment:
use arrow function syntax to keep things consistent: `notification.onclick = (event) => {`

Reply:
Fixed